read-fonts/src/lib.rs

```diff
@@ -60,6 +60,8 @@
 #![deny(rustdoc::broken_intra_doc_links)]
 #![cfg_attr(not(feature = "std"), no_std)]
 
+extern crate alloc;
```

Comment:
This shouldn't be necessary... if you have any code that requires allocation it should be gated on `std`.

Reply:
Yes, avar2 fundamentally requires allocation. Though I think we can cover 100% of real world fonts by attempting allocation on the stack first. An array of size 64 is probably of reasonable size. If you’d prefer not to add that here, I’ll do it as a follow up.

Reply:
My issue is more with our pattern for using `std` vs `alloc`; so far we've just been either `std` or not, and bringing in alloc sort of confuses that?

If we can just do `let mut buf = [F2Dot14::ZERO; 64];` and work in there that seems like a reasonable way to punt on `alloc` for now.

Reply:
Yeah, I can do that. The only reason I didn't is because my head was in "just implement the spec" mode, `axisCount` is a uint16, and that's a lot of bytes to put on the stack. Maybe we can do both - in a `no-std` environment, use a stack-allocated array and blow up if there are more than 64 axis: in `std`, DTRT with allocation.

Reply:
yea let's just do the 64 for the time being to get this merged, and then I we'll open an issue to figure out what the best long-term approach is.

Reply:
This sgtm as well. I assume most people are going to be using skrifa anyway so we can muck about with the read-fonts side to find something suitable.